app/javascript/packages/document-capture/components/acuant-camera.tsx

```diff
@@ -94,9 +94,9 @@ interface AcuantCameraUIOptions {
 }
 
 export enum AcuantDocumentType {
-  NONE = 0,
-  ID = 1,
-  PASSPORT = 2,
+  none = 0,
```

Comment:
I think it's conventional to use uppercase for enums but doing this in this way lets us simplify `getDocumentTypeLabel` a lot

Reply:
it looks like this is used in analytics only, would it be disruptive if we sent uppercase to the logs?

Reply:
Could we throw in a `toLowerCase` to get it in the expected case in `getDocumentTypeLabel`?

Reply:
Sure, went for it in c7aa3fa81cdf01fdeaf164fa0b7e8292a48069cf